tests/client_test.py

```diff
@@ -15,12 +15,41 @@
 
 
 import os
+import mock
+import yaml
+import json
 
 import google.ads.google_ads.client
 import google.ads.google_ads.v0
-import mock
+from google.ads.google_ads.v0.proto.services import google_ads_service_pb2
 from pyfakefs.fake_filesystem_unittest import TestCase
-import yaml
+
+
+class ModuleLevelTest(TestCase):
```

Comment:
If you're not using pyfakefs, probably doesn't make sense to use `pyfakefs.fake_filesystem_unittest.TestCase`.

Reply:
I updated this so that the file imports TestCase then I aliased the old TestCase to FileTestCase so it's only inherited by the class that uses it, all others bring in the basic TestCase from unittest. See this commit: 9ed51da471f8fd5155a9bf2acff926572a8c7635

Reply:
I updated this so that the logging functionality only fires if the logger is configured to log at `WARNING` or below. See these two commits:

5e25a651ff437f5bbe4b8638a38a412cbb2509ed
5f6abff1c27a5addde89a346427622e21d01e1a6